pkg/services/rpcsrv/subscription_test.go

```diff
@@ -625,3 +625,60 @@ func TestSubscriptionOverflow(t *testing.T) {
 	finishedFlag.CompareAndSwap(false, true)
 	c.Close()
 }
+
+func TestFilteredIsValidSubscriptions(t *testing.T) {
```

Comment:
`TestFilteredSubscriptions_InvalidFilter` (because that's exactly what you're testing) or `TestFilteredSubscriptions_FilterIsValid`?